### PR TITLE
Support multiple advanced indexing on ONNX export

### PR DESCRIPTION
**Pull request by disktnk**
Using `GatherND` operator, added from upset11. ~Please merge #8341 first.~ <- merged, thx

### CODE REVIEW

File: onnx_chainer/functions/array.py [hunk 1]
```diff
@@ -127,16 +139,32 @@ def convert_GetItem(func, opset_version, input_names, output_names, context):
...
-            if gather_axis:
-                raise ValueError(
-                    'ONNX-Chainer does not support multiple advanced index')
-            gather_axis.append(axis - len(squeeze_idxs) + len(unsqueeze_idxs))
-            if isinstance(idx, list):
-                gather_idx = np.array(idx, dtype=np.int64)
+            if prev_gathered_axis >= 0:
+                if (i-1) != prev_gathered_axis:
+                    raise ValueError(
+                        'ONNX-Chainer does not support split multiple advanced'
```
[the remark this answers is not in the document]
Comment (disktnk) [reply onnx_chainer/functions/array.py]: fixed, thanks for reviewing

**Comment by take-cheeze** [comment]
Jenkins test this please!

**Comment by take-cheeze** [comment]
Jenkins test this please

**Comment by take-cheeze** [comment]
Seems travis status isn't notified to github.
Merging

**Pull request merged**